src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -88,5 +91,38 @@ def generate_strategy_pool():
     pool.add_strategy(strategy.FORK_STRATEGY)
   if do_strategy(strategy.MUTATOR_PLUGIN_STRATEGY):
     pool.add_strategy(strategy.MUTATOR_PLUGIN_STRATEGY)
+  return pool
+
+
+def generate_strategy_pool():
+  """Generate a strategy pool based on probability
+  distribution from multi armed bandit experimentation."""
+  query = data_types.FuzzStrategyProbability.query()
+  distribution = list(ndb_utils.get_all_from_query(query))
```

Comment:
Do we need to force this to a list?

Reply:
I think so, I believe get_all_from_query returns an iterable through yield statements so I think it's a generator rather than a list